src/main/java/build/buildfarm/cas/BUILD

```diff
@@ -7,6 +7,7 @@ java_library(
         "@maven//:org_xerial_sqlite_jdbc",
     ],
     deps = [
+        "//src/main/java/build/buildfarm/backplane",
```

Comment:
It is odd to present the backplane interface to the CFC. It only makes use of a small part of it, and other instances where the CFC influences distributed content (i.e. the CAS index) have been accomplished with callbacks, and this bleeds the requirements for a backplane at all into the CAS.

Reply:
Thanks for pointing out. Agreed metrics package is more suitable for this. Moved.